src/gt4py/cartesian/frontend/nodes.py

```diff
@@ -199,6 +200,20 @@ class Domain(Node):
     parallel_axes = attribute(of=ListOf[Axis])
     sequential_axis = attribute(of=Axis, optional=True)
 
+    @classmethod
+    def from_gtscript(cls, gt_axis: list[gt_script.Axis]):
+        sequential_axis = None
+        parallel_axes = []
+        for axis in gt_axis:
+            if axis == "I" or "J":
```

Comment:
I think this should be
```suggestion
            if axis == "I" or axis == "J":
```
or
```suggestion
            if axis in ("I", "J"):
```